examples/Button/BasicButton.md

```diff
@@ -1,3 +1,3 @@
 ```jsx
-<BasicButton clickHandler={function(){alert('you clicked the button')}}>Click Me</BasicButton>
+<BasicButton clickHandler={function(){alert('You clicked the basic button')}}>Click Me!</BasicButton>
```

Comment:
Can we reformat these examples to meet our eslint guidelines? (i.e. parens after function, line-widths, jsx props on separate lines, etc.) I know eslint won't actually flag these as problems, but I think we should try to make our examples as copy-and-pastable as possible.

Reply:
Fixed through e59d270!